packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -465,49 +461,38 @@ def get(self, track_id):
         """
         request_args = stream_parser.parse_args()
         is_preview = request_args.get("preview")
-        filename = request_args.get("filename")
         decoded_id = decode_with_abort(track_id, ns)
-        info = get_track_stream_info(decoded_id)
+        info = get_track_access_info(decoded_id)
 
         track = info.get("track")
-        cid = track.get("track_cid")
-        if is_preview:
```

Comment:
how will track previews work?

Reply:
`"is_preview"` is passed into `get_track_stream_signature` and handled there